.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -5,7 +5,7 @@
 jobs:
 - job: win
   pool:
-    vmImage: windows-2019
+    vmImage: vs2017-win2016
```

Comment:
After trying to refresh PR ( https://github.com/conda-forge/cmake-feedstock/pull/152 ) to merge, noticed a simple re-render broke that rather trivial PR. One of the changes was to the image highlighted. Trying reverting to the old image here to see if that fixes the issue